src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/InitialPage.tsx

```diff
@@ -150,16 +149,16 @@ function WorkspaceRateAndUnitPage(props: WorkspaceRateAndUnitPageProps) {
 
 WorkspaceRateAndUnitPage.displayName = 'WorkspaceRateAndUnitPage';
 
-export default compose(
-    withOnyx<WorkspaceRateAndUnitPageProps, WorkspaceRateAndUnitOnyxProps>({
-        // @ts-expect-error: ONYXKEYS.REIMBURSEMENT_ACCOUNT is conflicting with ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM
-        reimbursementAccount: {
-            key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
-        },
-        workspaceRateAndUnit: {
-            key: ONYXKEYS.WORKSPACE_RATE_AND_UNIT,
-        },
-    }),
-    withPolicy,
-    withNetwork(),
-)(WorkspaceRateAndUnitPage);
+export default withNetwork()(
```

Comment:
I believe it's not needed since the `network` object is not used anywhere https://github.com/Expensify/App/blob/4400aca0b2948f8a2945e16335975152851df5b6/src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/InitialPage.tsx#L28-L31

this change was introduced [here](https://github.com/Expensify/App/pull/18454) but there is no reference for why it was introduced.

Reply:
Okay done